### ISSUE TITLE
Optimization & A11Y Issues

### ISSUE THREAD

**Issue by rmolinamir** [rmolinamir/personal#39]
- [x] Fix the tab order in the desktop, and windows (I think a flex-reverse is affecting the order).
- [x] Make the clock accessible.
- [x] Switching themes is a bit slow the first time, might be worth to look into SVGs/clip-path. Need to optimize for that. The theme toggle animation also shows in the bottom-right corner while the page is loading. It's probably being called on startup.
- [x] Windows should be activated on focus trap.
- [x] Fix tab strip tab order.
- [x] Add scroll into view to the taskbar applications whenever the windows become active.
- [x] If you hide the toolbar on mobile, the BSOD screen is buggy, the size isn't right. Power screen possibly too.
- [x] Apps take a while to load because they're lazy loaded, so I'll add a simple `cursor: wait` styling to the document body while the Suspense is active, via the application fallbacks.
- [x] Add SEO, meta tags, etc. See docs: [SEO | TanStack Start React Docs](https://tanstack.com/start/latest/docs/framework/react/guide/seo)
- [x] Sometimes, after playing DOOM for a while on mobile, the website becomes impossible to interact with, some buttons won't work, etc.
